.circleci/config.yml

```diff
@@ -109,14 +117,6 @@ jobs:
           command: |
             sh ./scripts/heroku.sh << parameters.service >> << parameters.app-name >>
 
-  open-prod-pull-request:
```

Comment:
Hum. Can we keep it for now? I think it's ok that it opens 2 prod deploys for now (we can still close one!)